docs/topics/deployment/manually-with-asgi-server.rst

```diff
@@ -0,0 +1,102 @@
+Manually with ASGI server
+==========
+
+ASGI (Asynchronous Server Gateway Interface) is intended to provide a standard interface between async Python web frameworks like Litestar, and async web servers such as the ones below.
+
+Uvicorn
+-------
+
+`Uvicorn <https://www.uvicorn.org/>`_ is an ASGI server that supports HTTP/1.1 and WebSocket.
+
+Hypercorn
+---------
+
+`Hypercorn <https://hypercorn.readthedocs.io/en/latest/#/>`_ is an ASGI server that was initially part of `Quart <https://pgjones.gitlab.io/quart//>`_, and supports HTTP/1.1, HTTP/2, and WebSocket.
+
+Daphne
+------
+
+`Daphne <https://github.com/django/daphne/>`_ is an ASGI server that was originally developed for `Django Channels <https://channels.readthedocs.io/en/latest/>`_, and supports HTTP/1.1, HTTP/2, and WebSocket.
+
+Granian
+-------
+
+`Granian <https://github.com/emmett-framework/granian/>`_ is a Rust based ASGI server that supports HTTP/1.1, HTTP/2, and WebSocket.
+
+Install the ASGI Server
+-----------------------
+.. tab-set::
+
+    .. tab-item:: Uvicorn
+
+        .. code-block:: shell
+
+            pip install uvicorn
+
+    .. tab-item:: Hypercorn
+
+        .. code-block:: shell
+
+            pip install hypercorn
+
+    .. tab-item:: Daphne
+
+        .. code-block:: shell
+
+            pip install daphne
+
+    .. tab-item:: Granian
+
+        .. code-block:: shell
+
+            pip install granian
```

Comment:
These and all following and previous tab sets can use the sync to change each tab to the value

- https://sphinx-design.readthedocs.io/en/latest/tabs.html#synchronised-tabs if you are using sphinx-design (preffered)
- https://sphinx-tabs.readthedocs.io/en/latest/#group-tabs if you are using sphinx-tabs (should be removed as a dep. tbh)

Reply:
thank you for the suggestion! 🙏 
done in https://github.com/litestar-org/litestar/pull/2663/commits/e97a9ea00bd2311a793531b49c8839423b41772f